uqcsbot/scripts/news.py

```diff
@@ -0,0 +1,39 @@
+from uqcsbot import bot, Command
+from uqcsbot.utils.command_utils import loading_status
+from typing import Tuple, Dict, List
+
+import feedparser
+
+ARTICLES_TO_POST = 5
+URL = "http://feeds.feedburner.com/TechCrunch/"
+
+def get_tech_crunch_data() -> List[Dict[str, str]]:
+    """
+    Returns data from TechCrunch RSS feed
+    """
+    return feedparser.parse(URL).entries
+
+def get_data_from_article(news: List[Dict[str, str]], index: int) -> Tuple[str, str]:
+    """
+    Returns the title of the article and the link
+
+    Tuple returned: (title, url)
+    """
+    return (news[index]['title'], news[index]['link'])
+
+@bot.on_command("news")
+@loading_status
+def handle_news(command: Command) -> None:
+    """
+    Prints the 5 top-most articles in the Latest News Section of TechCrunch
+    using RSS feed
+    """
+    message = "------------------------- Latest News from TechCrunch ---------------------------\n"
```

Comment:
Formatting nit, can the heading be bold like `**Example Title**` instead? A link to techcrunch.com would also be nice.

Reply:
How do you enable Markdown formatting so that the heading is bold. Simply using ** does not seem to work for me

Reply:
My bad, Slack uses single asterisks for bold: https://api.slack.com/reference/surfaces/formatting